### ISSUE TITLE
Add a test on App.vue

### ISSUE THREAD

**Issue by trollepierre** [eddyerburgh/vue-test-utils-vuex-example#7]
In order to see how to test parent component that are shallowed and without store...
Because I didn't succeed it

**Comment by eddyerburgh** [comment]
Did you use the `shallow` method to mount App?

**Comment by trollepierre** [comment]
Of course, but it appears there are side effects between Vue Test Utils, Vuex...
when a child component starts using the store, the parent needs to use(Vuex). 
when I shallow

**Comment by eddyerburgh** [comment]
Can you post a reproduction? This sounds like a bug we should fix
